basyx.aasxfileserver/basyx.aasxfileserver-core/pom.xml

```diff
@@ -17,7 +17,10 @@
 			<groupId>org.eclipse.digitaltwin.basyx</groupId>
 			<artifactId>basyx.core</artifactId>
 		</dependency>
-
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-mongodb</artifactId>
```

Comment:
This should not be in core

Reply:
This needs to be in core, as the Package.java needs Annotations from this Package to correctly handle MongoDB Field mapping